Makefile

```diff
@@ -12,7 +12,7 @@ clean:
 	rm matchup/sim/*/*.class
 
 run:
-	java matchup.sim.Simulator -p random random
```

Comment:
Can you please revert the change to this file? I will merge your changes after that.

Reply:
Similarly for Random.java and History.java.